src/scheduler/globals.h

```diff
@@ -41,7 +41,7 @@ extern "C" {
 #endif
 
 #include "data_types.h"
-
+#include "limits.h"
```

Comment:
Why is this needed in globals.h?

Reply:
This header is needed to get the definition of _POSIX_PATH_MAX